### ISSUE TITLE
'Screen Reader detected' notification permanently on bottom bar

### ISSUE THREAD

**Issue by euk-anthonyb** [Microsoft/vscode#29752]
- VSCode Version: Code 1.13.1 (379d2efb5539b09112c793d3d9a413017d736f89, 2017-06-14T18:21:47.485Z)
- OS Version: Windows_NT ia32 10.0.10586
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|fresh-material|2ndshift|2.2.2|
|Bookmarks|alefragnani|0.15.2|
|project-manager|alefragnani|0.18.1|
|htmltagwrap|bradgashler|0.0.3|
|gotemplate|casualjim|0.1.1|
|bracket-pair-colorizer|CoenraadS|0.10.7|
|mustache|dawhite|1.0.0|
|gitlens|eamodio|4.2.0|
|vscode-html-css|ecmel|0.1.7|
|json-tools|eriklynd|1.0.2|
|beautify|HookyQR|1.1.1|
|automagic-code|jairzjunior|0.0.2|
|theme-material-dark-soda|jbw91|1.2.0|
|theme-atomic-dark|kimkwanka|0.2.0|
|git-indicators|lamartire|1.2.1|
|vs-color-picker|lihui|0.6.0|
|start-git-bash|McCarter|1.2.1|
|code-beautifier|michelemelluso|1.7.2|
|prettify-json|mohsen1|0.0.3|
|vscode-scss|mrmlnc|0.6.2|
|atom-keybindings|ms-vscode|2.9.0|
|debugger-for-chrome|msjsdiag|3.1.6|
|subtle-brackets|rafamel|2.0.0|
|vscode-icons|robertohuertasm|7.10.1|
|vscode-theme-superonedark|seansassenrath|0.0.15|
|code-settings-sync|Shan|2.8.1|
|vscode-todo-highlight|wayou|0.5.5|
|vscode-material-syntax-dark|woodywoodsta|0.1.1|
|Material-theme|zhuangtongfa|2.8.9|;

---

Steps to Reproduce:

1. Launched NVDA on Windows (10) yesterday afternoon
2. Ever since that moment, the screen reader notification is permanently on the bottom bar, even after quitting the screen reader and restarting my PC several times
3. In addition, my issues with lines not wrapping and the mini-map disappearing (which I only just noticed) seem to be related to this. 

![image](https://user-images.githubusercontent.com/14346320/27638012-a4f5b6ce-5c09-11e7-95ea-a503a214d647.png)

:(

**Comment by alexandrudima** [comment]
@euk-anthonyb 

Electron enters this mode when a Screen Reader is actively communicating with it (i.e. it is not persisted between VS Code restarts). On Windows, Electron does not detect when a Screen Reader stops communicating with it, so closing NVDA will leave VS Code in this mode. But restarting VS Code should result in this mode being left. Perhaps NVDA is still running? More information [here](https://code.visualstudio.com/updates/v1_13#_screen-reader-detected).

In any case, since you're probably using NVDA to test other software and do not wish VS Code to become optimized for screen readers, hover over the status bar entry to find out how to disable this mode:
<img width="646" alt="screen shot 2017-06-29 at 09 34 43" src="https://user-images.githubusercontent.com/5047891/27676441-342bb9ca-5cae-11e7-8b29-b7a7d81867c4.png">

Do you have any UX improvements about how to make it more clear that there exists a setting to turn this support off? I thought people would hover over the status bar entry to find out more about it.

**Comment by alexandrudima** [comment]
I'm thinking we should show a big pop-up on top of the status bar item that explains what's going on.

**Comment by euk-anthonyb** [comment]
Hi Alexandru,

Thanks for your help. I've now adjusted my settings and the editor is
working as normal once more. :)

I was clicking on the green button like crazy but nothing was happening. It
didn't occur to me that I should hover over it instead.

I would suggest that a larger pop-up automatically appears when a screen
reader is encountered, containing instructions on what to do next and then
perhaps another reminder pop-up when the status bar notification label is
clicked.

I wasn't aware that there were accessibility settings until I went hunting
for them.


*Anthony Browne*
Senior Designer

Eukleia Training Limited
Sherborne House, 5th Floor, 119-121 Cannon Street, London EC4N 5AT

*t*   +44 (0)2072204057 (direct)
    +44 (0)2072204050 (general)
*f*   +44 (0)2072204051

*e* anthony.browne@eukleia.com
*w* eukleia.com
<http://eukleia-training.com>

​
We use *Google* Hangouts
